scss/_accordion.scss

```diff
@@ -58,39 +57,40 @@
 }
 
 .accordion-item {
+  margin-bottom: -$accordion-border-width;
```

Comment:
following lines change the look and feel of our accordions 
https://deploy-preview-650--boosted.netlify.app/docs/5.0/components/accordion/
https://v5-dev--boosted.netlify.app/docs/5.0/components/accordion/

Reply:
Accordion has been re-accorded to Orange's look & feel (b547626)

Reply:
ok